courses/api.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: TD002, TD003, FIX002
```

Comment:
We'll want to remove these before I land it. This is just so I can put todos in for development